db/dcrpg/pgblockchain.go

```diff
@@ -5828,6 +5829,23 @@ func (pgb *ChainDB) GetExplorerBlock(hash string) *exptypes.BlockInfo {
 		Subsidy:               pgb.BlockSubsidy(b.Height, b.Voters),
 	}
 
+	if pgb.IsDCP0011Active(b.Height) {
+		blockChainHash, err := chainhash.NewHashFromStr(data.Hash)
+		if err != nil {
+			log.Errorf("error parsing hash for block %d (hash=%s): %v", data.Height, data.Hash, err)
+			return nil
+		}
+
+		// Get the block header
+		header, err := pgb.Client.GetBlockHeader(pgb.ctx, blockChainHash)
+		if err != nil {
+			log.Errorf("failed to fetch header for block hash %s: %v", data.Hash, err)
+			return nil
+		}
```

Comment:
This looks a bit expensive?

Reply:
Ideally `PowHash` would be added to `chainjson.GetBlockVerboseResult`.  We get this just above, at the start of this method.  AFAIK, we can only get this at present by hashing the block header.

Reply:
Would require: https://github.com/decred/dcrd/pull/3154